app/lib/frontend/templates/views/shared/layout.dart

```diff
@@ -167,6 +167,10 @@ d.Node pageLayoutNode({
             if (activeConfiguration.isStaging) 'staging-banner',
           ],
           children: [
+            // The initialization of dark theme must be in a synchronous, blocking
+            // script execution, as otherwise users may see flash of unstyled content
+            // (usually white background instead of a dark theme).
+            d.script(src: staticUrls.getAssetUrl('/static/js/dark-init.js')),
```

Comment:
Just to confirm: `/static/` is served with caching headers, meaning that this file will not be fetched over the network on most page loads?

Reply:
Yes. `getAssetUrl` rewrites the URL to use hashes and as long as we are on the same hash it will be served from cache.